packages/async_ssl/async_ssl.v0.9.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "opensource@janestreet.com"
+authors: "Jane Street Group, LLC <opensource@janestreet.com>"
+homepage: "https://github.com/janestreet/async_ssl"
+bug-reports: "https://github.com/janestreet/async_ssl/issues"
+license: "Apache-2.0"
+dev-repo: "https://github.com/janestreet/async_ssl.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "async" {>= "v0.9" & < "v0.10"}
+  "base" {>= "v0.9" & < "v0.10"}
+  "configurator" {>= "v0.9" & < "v0.10"}
+  "core" {>= "v0.9" & < "v0.10"}
+  "jbuilder" {build & >= "1.0+beta11"}
+  "ppx_driver" {>= "v0.9" & < "v0.10"}
+  "ppx_jane" {>= "v0.9" & < "v0.10"}
+  "stdio" {>= "v0.9" & < "v0.10"}
+  "conf-openssl"
+  "ctypes"
```

Comment:
I'm fairly sure this needs a minimum bound on ctypes of at least >=0.3 due to using cstubs